### DISCUSSION TITLE
translating NextJS documentation

### DISCUSSION THREAD

**Discussion by techhtml** [vercel/next.js#36603] [Ideas]
I have suggestion about official NextJS document.

Me and some others translating official NextJS document into Korean, 
Because I don't know how to build the docs, 
I think there are no ways to contributing.

But we just translating it for better community.

Here is my suggestion:
1. translate the pages, and build it.
2. deploy it to my own sites (maybe vercel)
3. Share the documents URL to Vercel.

It's fit for us?

**Comment by SeungrokYoon** [comment]
I have the same suggestion, and I've wondered how I could contribute translating the Next.JS page into Korean. I couldn't find any ways to contribute to Next.JS regarding documentation translating issue.
